tests/models/test_models.py

```
@@ -0,0 +1,565 @@
+
+    id = factory.Sequence(lambda n: n)
+    name = factory.Sequence(lambda n: f"Path {n}")
+    source_path = factory.Sequence(lambda n: f"/srv/hot_data_{n}")
+    operation_type = OperationType.MOVE
+    check_interval_seconds = 3600
+    enabled = True
+    prevent_indexing = True
+    last_scan_status = ScanStatus.SUCCESS
+
+    @factory.post_generation
+    def storage_locations(self, create, extracted, **kwargs):
```

Ruff (0.14.14)

[warning] Missing type annotation for `**kwargs`

(ANN003)